insights/parsers/openshift_get.py

```diff
@@ -40,6 +49,9 @@
 OcGetService - command ``oc get service -o yaml --all-namespaces``
 ------------------------------------------------------------------
 
+OcGetEgressNetworkPolicy - command ``oc get egressnetworkpolicy -o yaml --all-namespaces``
+------------------------------------------------------------------------------------------
+
```

Comment:
These subheadings should be in sorted order since they show up in the documentation in the same order they appear in this file.